src/Features/Core/Portable/UseAutoProperty/AbstractUseAutoPropertyCodeFixProvider.cs

```diff
@@ -46,10 +46,15 @@ public sealed override Task RegisterCodeFixesAsync(CodeFixContext context)
         {
             foreach (var diagnostic in context.Diagnostics)
             {
+                var priority = diagnostic.Properties.ContainsKey("LowPriority")
```

Comment:
❓ Is there a reason this doesn't use the approach of `UseExpressionBodyCodeFixProvider` (i.e. setting to low priority when the diagnostic severity is hidden)?

Reply:
nope.  that seems reasonable :)

Reply:
done.